File: sdks/insuretech-go-sdk/pkg/models/underwriting_health_declaration.go
package models

import (
	"time"
)

// UnderwritingHealthDeclaration represents a underwriting_health_declaration
type UnderwritingHealthDeclaration struct {
	AlcoholConsumer bool `json:"alcohol_consumer,omitempty"`
	AuditInfo interface{} `json:"audit_info"`
	Bmi string `json:"bmi,omitempty"`
	FamilyHistory string `json:"family_history,omitempty"`
	HasFamilyHistory bool `json:"has_family_history,omitempty"`
	HasPreExistingConditions bool `json:"has_pre_existing_conditions,omitempty"`
	HeightCm int `json:"height_cm,omitempty"`
	Id string `json:"id"`
	IsCurrentlyHospitalized bool `json:"is_currently_hospitalized,omitempty"`
	MedicalDocuments string `json:"medical_documents,omitempty"`
	MedicalExamCompleted bool `json:"medical_exam_completed,omitempty"`
	MedicalExamDate time.Time `json:"medical_exam_date,omitempty"`
	MedicalExamRequired bool `json:"medical_exam_required,omitempty"`
	MedicalExamResults string `json:"medical_exam_results,omitempty"`
	OccupationRiskLevel string `json:"occupation_risk_level,omitempty"`
	PreExistingConditions string `json:"pre_existing_conditions,omitempty"`
	QuoteId string `json:"quote_id"`
	Smoker bool `json:"smoker,omitempty"`
	WeightKg string `json:"weight_kg,omitempty"`
}

File: backend/insurance_engine-old/src/InsuranceEngine.Underwriting/Domain/Entities/UnderwritingHealthDeclaration.cs
using System;
using InsuranceEngine.Underwriting.Domain.Enums;

namespace InsuranceEngine.Underwriting.Domain.Entities;

public class UnderwritingHealthDeclaration
{
    public Guid Id { get; set; }
    public Guid QuoteId { get; set; }

    public int HeightCm { get; set; }
    public decimal WeightKg { get; set; }
    public decimal Bmi { get; set; }

    public bool HasPreExistingConditions { get; set; }
    public string? PreExistingConditionsJson { get; set; }
    public bool IsCurrentlyHospitalized { get; set; }
    public bool HasFamilyHistory { get; set; }
    public string? FamilyHistoryJson { get; set; }

    public bool IsSmoker { get; set; }
    public bool IsAlcoholConsumer { get; set; }
    public string? OccupationRiskLevel { get; set; }

    public bool IsMedicalExamRequired { get; set; }
    public bool IsMedicalExamCompleted { get; set; }
    public string? MedicalExamResultsJson { get; set; }
    public DateTime? MedicalExamDate { get; set; }

    public string? MedicalDocumentsJson { get; set; }

    public string? AuditInfoJson { get; set; }
    
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
}
